batch-processing/README.md

```diff
@@ -0,0 +1,379 @@
+# Batch Processing
+
+In this project, you'll build a simple image processing pipeline: reading from a CSV file, downloading an image, processing it, and uploading it to cloud storage.
+
+> ⚠️ This project requires you to have access to an Amazon AWS account, with permissions to configure IAM and S3. Ask on CYF Slack for help with that.
+
+> ⚠️ You need a GitHub repo to complete this because we'll be using [GitHub Actions](https://docs.github.com/en/actions).
+
+Learning objectives:
+
+- What is batch processing and how does it differ from building servers?
+- How do we build resilience into batch processing software?
```

Comment:
I'm not sure we've quite gotten this one yet :)

Reply:
No indeed.

Reply:
I've now added some pointers & extensions. We can return to this project later to tweak and improve